src/main/java/com/mongodb/shardsync/command/sync/SyncMetadataCommand.java

```diff
@@ -0,0 +1,55 @@
+package com.mongodb.shardsync.command.sync;
+
+import com.mongodb.shardsync.ShardConfigSync;
+import com.mongodb.shardsync.SyncConfiguration;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.util.concurrent.Callable;
+
+@Command(name = "metadata", description = "Synchronize shard metadata between clusters")
+public class SyncMetadataCommand implements Callable<Integer> {
+    
+    @CommandLine.ParentCommand
+    private SyncCommand parent;
+    
+    @Option(names = {"--optimized"}, 
+            description = "Combine adjacent chunks for optimization (default: true)", 
```

Comment:
Since most flags here default to off, would it make more sense for the flag to be `--skipOptimizeAdjacent` or something?